.github/workflows/jit-format.yml

```diff
@@ -29,13 +35,13 @@ jobs:
           ref: main
           token: ${{ secrets.GITHUB_TOKEN }}
       - name: Install .NET
-        uses: actions/setup-dotnet@v1
+        uses: actions/setup-dotnet@v4
         with:
           dotnet-version: '8.0.x'
       - name: Build jitutils
         run: |
-          cd jitutils
           ./bootstrap${{ matrix.os.extension }}
+        working-directory: jitutils
       - name: Checkout runtime
         uses: actions/checkout@v4
         with:
```

Comment:
I think we can remove with: path: runtime, to reduce nesting ` Initialized empty Git repository in /__w/runtime/runtime/runtime/.git/`

Reply:
I want to make sure we have separate side-by-side paths for jitutils and runtime to avoid possible issues with cloning jitutils into a subdirectory in dotnet/runtime. This is the recommended way to do so as far as I saw.

Reply:
Ah right. Without it, it uses current directory. Was wondering where are other two `runtime/` path slugs coming from. 🤔

Reply:
We should be in `working-directory: runtime` for this task for jituitls to run the git command. The bare git provides this option `git -C some/other/path diff` etc. but I think we can't control it what jitutils internally invoke (they haven't exposed this feature).

Reply:
The git command that produces the patch file is run in the `jitformat.py` script, not jitutils. I've updated it to invoke in the passed-in runtime directory (which is technically required to get the correct result).